### ISSUE TITLE
SS_doRetro won't run with 3.30.01 model

### ISSUE THREAD

**Issue by k-doering-NOAA** [r4ss/r4ss#458]
working on issue #448 made me realize that 3.30.01 can't run with SS_doRetro (see [this test](https://github.com/r4ss/r4ss/blob/development/tests/testthat/test-runs.R#L33-L62)) . At first, I thought it was because I was running it with a 3.30.16 version of Stock Synthesis, but testing locally with a [3.30.01 version of stock synthesis](https://vlab.ncep.noaa.gov/web/stock-synthesis/document-library/-/document_library/0LmuycloZeIt/view/13078378) also caused the tests to fail.

I'm not sure if it is worth trying to fix this, if it is a bug with stock synthesis and not with r4ss. The other versions of the same model input files work fine, so I think if this situation affects any users, upgrading to the latest version of stock synthesis will likely solve the problem.

**Comment by iantaylor-NOAA** [comment]
I ran the individual pieces of the test on my computer and couldn't replicate the error. I suspect that 3.30.01 is not worth supporting and we should remove it from the tests following the plan in https://github.com/r4ss/r4ss/issues/451. I think the only thing that made 3.30.01 worthy of including in the tests is that it was the current version in April 2017 when testthat tests were first added to r4ss.

**Comment by k-doering-NOAA** [comment]
huh, @iantaylor-NOAA I'm not sure why this error isn't replicable...

I looked into this a bit more, and it was definitely an issue with stock synthesis itself - changing the retro year from 0 to -1 in the 3.30.01 model was enough to cause an error for me when I tried to run it with ss 3.30.01. Since this problem isn't fixable in r4ss, I don't think we should be running this test (also, I can't find a linux version of 3.30.01 to run them with...)

I've just commented out the 3.30.01 tests in test-runs.  We may want to just delete them at some point.

I also added a warning message in SS_doRetro if there is no report file found in the folder for the retrospective run, which I think should indicate when the run didn't occur at all (although it is possible that the model didn't run all the way through and a Report file would be generated), so would help the user identify retro runs that failed. 

I'm closing this issue for now, but it can be reopened if these changes made aren't satisfactory.
